### DISCUSSION TITLE
final model in ert field data

### DISCUSSION THREAD

**Discussion by RFauziahSM** [gimli-org/pyGIMLi#351] [Q&A] [answered]
Hi, everyone.
Currently I'm still learning about pygimli, so please forgive my lack of knowledge about python.
I want to ask, in the following example [https://www.pygimli.org/_examples_auto/3_dc_and_ip/plot_02_ert_field_data.html#sphx-glr-examples-auto-3-dc-and-ip-plot-02-ert-field-data-py](url)
why the resulting model has a resistivity model at a distance before 0?
is the model at the distance before 0 an inversion model of the field data?

**Comment by halbmy** [comment, marked as answer]
The model discretization typically (also in other inversion programs) includes also regions outside the electrode line just in case some contrast happens there. For the forward calculation we need values outside. The sensitivity distribution (https://www.pygimli.org/_examples_auto/3_dc_and_ip/plot_05_ert_4_point_sensitivities.html) also extends beyond the electrodes so the inversion domain should cover this.

Generally there is no harm to choose the inversion domain rather too large (resulting in regions of weak coverage that are filled by smoothness constraints) instead of too small (resulting in bad data fit or mapping of anomalies into inside regions).
